llvm/lib/CodeGen/SelectionDAG/SelectionDAGBuilder.cpp

```diff
@@ -10446,8 +10446,8 @@ TargetLowering::LowerCallTo(TargetLowering::CallLoweringInfo &CLI) const {
     SmallVector<EVT, 4> ValueVTs;
     SmallVector<Type *, 4> IRTypes;
     TypeSize Offset = TypeSize::get(0, Args[i].Ty->isScalableTy());
-    ComputeValueVTs(*this, DL, Args[i].Ty, ValueVTs, /*Offsets=*/nullptr,
-                    Offset, &IRTypes);
+    ComputeValueVTs(*this, DL, Args[i].Ty, ValueVTs, /*MemVTs*/ nullptr,
+                    /*Offsets=*/nullptr, Offset, &IRTypes);
```

Comment:
Note: If we drop AIE1 support, those changes we made here can go. I think the `IRTypes` parameter is something I added a long time ago to identify pointer types.

Reply:
Yes, GlobalISel CallLowering tracks pointer type/address spaces separately nowadays, so this change is only needed for SelectionDAG